beacon-chain/sync/subscriber_sync_committee_message.go

```diff
@@ -9,6 +9,7 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+// skipcq: SCC-U1000
```

Comment:
What is this?

Reply:
nvm, its a deepsource thing

Reply:
Unused code to satisfy deep source for now, probably can be removed